packages/unity_video_player/unity_video_player_media_kit/lib/unity_video_player_media_kit.dart

```diff
@@ -107,6 +101,17 @@ class __MKVideoState extends State<_MKVideo> {
 
 class UnityVideoPlayerMediaKit extends UnityVideoPlayer {
   Player mkPlayer = Player();
+  late Future<VideoController> mkVideoController;
```

Comment:
`VideoController` should be free'd in `dispose` (ideally before `Player` using `await`).